addons/stock/models/stock_move.py

```diff
@@ -180,6 +180,8 @@ def _default_group_id(self):
     forecast_availability = fields.Float('Forecast Availability', compute='_compute_forecast_information', digits='Product Unit of Measure')
     forecast_expected_date = fields.Datetime('Forecasted Expected date', compute='_compute_forecast_information')
     lot_ids = fields.Many2many('stock.production.lot', compute='_compute_lot_ids', inverse='_set_lot_ids', string='Serial Numbers', readonly=False)
+    reservation_date = fields.Datetime('Date to Reserve', compute='_compute_reservation_date', store=True,
```

Comment:
Shouldn't be fields.Date?

Reply:
I was trying to make sure I matched the time in the move's `date` (i.e. "Date Scheduled"), is it better if I only the date?

Reply:
The issue is the scheduler runs once a day. So if the reservation is planned 1h after scheduler it won't be reserve for the correct day.

Reply:
Changed it

Reply:
![](https://media.giphy.com/media/ANbD1CCdA3iI8/giphy.gif)

Reply:
I swear I changed it before...

![](https://media.giphy.com/media/woWQA2I7xqRMI/giphy.gif)

Changed it (again?)